docs/internals/contributing/writing-code/unit-tests.txt

```diff
@@ -213,6 +213,22 @@ Going beyond that, you can specify an individual test method like this:
 
    $ ./runtests.py --settings=path.to.settings i18n.tests.TranslationTests.test_lazy_objects
 
+Running tests from specific point
+---------------------------------
```

Comment:
I don't think that a separate section is necessary here, keeping this under `Running only some of the tests` looks good for me.